src/main/host/process.rs

```diff
@@ -111,7 +111,7 @@ impl From<ThreadId> for ProcessId {
 #[derive(Debug, Copy, Clone, Eq, PartialEq)]
 pub enum ExitStatus {
     Normal(i32),
-    Signaled(nix::sys::signal::Signal),
+    Signaled(Signal),
```

Comment:
Do we show this value to the user? For example if the process fails with an unexpected exit state? If so, should we add a nicer debug impl to the linux-api Signal to show the signal name and not just the signal number?

Reply:
We don't. We convert to the configuration's `ProcessFinalState` and display that, so that the actual final state is displayed in a way that's consistent with how the expected final state is specified.